src/fiatconnect/saga.test.ts

```diff
@@ -83,27 +103,60 @@ describe('Fiatconnect saga', () => {
       )
         .provide([
           [select(userLocationDataSelector), { countryCodeAlpha2: 'MX' }],
-          [select(currentAccountSelector), '0xabc'],
           [select(getLocalCurrencyCode), 'USD'],
           [select(fiatConnectCashInEnabledSelector), false],
           [select(fiatConnectCashOutEnabledSelector), true],
+          [select(fiatConnectProvidersSelector), mockFiatConnectProviderInfo],
         ])
         .put(fetchFiatConnectQuotesFailed({ error: 'Could not fetch providers' }))
         .run()
 
       expect(fetchQuotes).toHaveBeenCalledWith({
-        account: '0xabc',
         country: 'MX',
         cryptoAmount: 3,
         digitalAsset: 'CELO',
         fiatConnectCashInEnabled: false,
         fiatConnectCashOutEnabled: true,
         flow: CICOFlow.CashIn,
         localCurrency: 'USD',
+        fiatConnectProviders: mockFiatConnectProviderInfo,
       })
     })
   })
 
+  describe('handles fetching providers', () => {
+    it('saves on success', async () => {
+      mocked(getFiatConnectProviders).mockImplementation(() =>
```

Comment:
you can use mockResolvedValue / mockRejectedValue as shorthands for mockImplementation